README.md

```diff
@@ -188,3 +188,24 @@ linkchecker --check-extern target/*.html
 ```
 
 If you remove the command line switch `--check-extern` only local links are checked. 
+
+## Continuous Integration
+
+This project contains a CI setup for Jenkins. The build uses docker to provide a stable environment for its execution. So Jenkins needs to be able to execute docker commands.
```

Comment:
Please change "docker" to `[Docker](https://www.docker.com/)`. First time we mention external source, we should link them.